main/test/src/mill/main/MainModuleTests.scala

```diff
@@ -93,6 +99,16 @@ object MainModuleTests extends TestSuite {
           res.contains("hello")
         )
       }
+      test("worker") - UnitTester(mainModule, null).scoped { eval =>
+        val Right(result) = eval.apply("inspect", "helloWorker")
+
+        val Seq(res: String) = result.value
+        assert(
```

Comment:
Would be good to also cover the scaladoc of the worker.

Reply:
Sure, why not. I had copied the tests for commands, that don't test the scaladoc.